.just/maven.just

```diff
@@ -3,7 +3,7 @@ default_target   := env('MVN_TARGET',   "verify")
 default_flags    := env('MVN_FLAGS',    "--threads 2C")
 skip_tests_flags := default_flags + " -DskipTests"
 
-default_verify_profiles := env('MVN_VERIFY_PROFILES', "errorprone,maven-enforcer-plugin,maven-dependency-plugin,maven-javadoc-plugin")
+default_verify_profiles := env('MVN_VERIFY_PROFILES', "errorprone-strict,maven-enforcer-plugin,maven-dependency-plugin,maven-javadoc-plugin")
```

Comment:
Most maven profiles work together but errorprone is tricky and I couldn't figure out a way to run separate profiles like we do for Checkstyle. I can run errorprone-strict or errorprone, but not both profiles together.